python/pyspark/ml/remote/proto.py

```diff
@@ -0,0 +1,86 @@
+#
```

Comment:
We're a bit in a bad position here because, high-level we should use the `pyspark.ml.connect` module, but this is occupied by the previous approach of the implementation.

I'm wondering if we should do some cleanup later to delete the previous code and move this one to the right package. 

@WeichenXu123 what do you think?

Reply:
agreed that we'd better move it to the right package